Makefile

```diff
@@ -61,9 +56,6 @@ endif
 test-unit-cov: clean sanitize build
 	hooks/coverage.sh
 
-test-integration: clean build
```

Comment:
Why is it OK to remove integration tests? Do they test only the metrics API?

Reply:
Yes.